partials/home-splash.php

```diff
@@ -0,0 +1,12 @@
+<?php
+$home = get_page_by_path('home');
+$splash_text = get_post_meta($home->ID,'_igv_splash_text', true);
```

Comment:
I think for themes which depend on certain pages existing we should do init checks and create the pages programatically: https://developer.wordpress.org/reference/functions/wp_insert_post/#parameters